leasing/management/commands/attach_areas.py

```diff
@@ -15,38 +17,47 @@
     PlotDivisionState,
 )
 
+logging.basicConfig(level=logging.INFO, format="%(message)s", stream=sys.stdout)
```

Comment:
Calling `basicConfig` should be done in the `handle` method rather than import time.